### PR TITLE
Fix missing icons/backgrounds

### LINKED ISSUE #271: Fix missing icons/backgrounds
The warning/error icons and backgrounds went missing. I have a feeling this relates to #225

### PR DESCRIPTION
**Pull request by linclark**
Fixes #271 

@bgrins this is ready for review

This leaves a lot to be done yet, but I think it gets all of the icons and backgrounds that we need in place. Once we can develop in a tab, we probably want to do a full redesign / css rewrite.

### CODE REVIEW

File: devtools/client/webconsole/new-console-output/components/message-types/console-api-call.js [hunk 1]
```diff
@@ -87,13 +87,9 @@ function ConsoleApiCall(props) {
...
   const classes = ["message", "cm-s-mozilla"];
 
-  if (source) {
-    classes.push(source);
-  }
-
-  if (level) {
-    classes.push(level);
-  }
+  classes.push(source);
```
Comment (bgrins) [inline devtools/client/webconsole/new-console-output/components/message-types/console-api-call.js:90]: source/type/level are always going to be non-falsy here?

File: devtools/client/themes/webconsole.css [hunk 2]
```diff
@@ -54,12 +55,14 @@ a {
 }
 
 .theme-firebug .message[severity="error"],
+.theme-light .message.error,
 .theme-firebug .message.error {
```
Comment (bgrins) [inline devtools/client/themes/webconsole.css:59]: How was this matching anything before?  With the old frontend we weren't adding the 'error' / 'warn' classes, right?

Comment (bgrins) [reply devtools/client/themes/webconsole.css:59]: Oh, I guess it was matching this rule instead for old frontend but we must have added in the support for the new classes earlier: `.theme-firebug .message[severity="error"],`

Comment (linclark) [reply devtools/client/themes/webconsole.css:59]: In old-console, we didn't have a background (AFAICT) for these messages in light theme. Instead we used the indent color.

I think having a background makes more sense, and it's what Chrome does. LMK if you think we should switch it.

**Review by bgrins — APPROVED** [review]
This works for me if this is correct: https://github.com/devtools-html/gecko-dev/pull/288#r79479814

File: devtools/client/webconsole/new-console-output/components/message-types/console-api-call.js (hunk 1 above)
Comment (linclark) [reply devtools/client/webconsole/new-console-output/components/message-types/console-api-call.js:90]: yes

**Pull request merged**